package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "4.0.1",
+  "version": "4.0.2",
```

Comment:
Technically this should be a minor version change, since you added new functionality